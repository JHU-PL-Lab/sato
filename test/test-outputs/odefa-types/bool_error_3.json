[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "x", "i" ],
          "err_match_val": "input",
          "err_match_expected": "bool",
          "err_match_actual": "int"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "ret = x and tru",
    "err_steps": 388
  }
]