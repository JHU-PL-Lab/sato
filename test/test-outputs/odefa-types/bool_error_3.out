Type errors for:
- Input sequence  : [0]
- Found at clause : ret = x and tru
--------------------
* Value    : x = i = input
* Expected : bool
* Actual   : int
1 errors found. No further control flows exist.
